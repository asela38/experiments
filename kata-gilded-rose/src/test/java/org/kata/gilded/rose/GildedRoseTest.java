package org.kata.gilded.rose;

import java.util.Arrays;
import java.util.LinkedList;
import java.util.List;

import org.approvaltests.Approvals;
import org.junit.Test;

public class GildedRoseTest {

	@Test
	public void approvalVerifyAll() {
		Item[] items = new Item[] { new Item("+5 Dexterity Vest", 10, 20), //
				new Item("Aged Brie", 2, 0), //
				new Item("Elixir of the Mongoose", 5, 7), //
				new Item("Sulfuras, Hand of Ragnaros", 0, 80), //
				new Item("Sulfuras, Hand of Ragnaros", -1, 80),
				new Item("Backstage passes to a TAFKAL80ETC concert", 15, 20),
				new Item("Backstage passes to a TAFKAL80ETC concert", 10, 49),
				new Item("Backstage passes to a TAFKAL80ETC concert", 5, 49),
				// this conjured item does not work properly yet
				new Item("Conjured Mana Cake", 3, 6) };

		GildedRose app = new GildedRose(items);

		int days = 50;

		List<Item> itemList = new LinkedList<>();
		for (int i = 0; i < days; i++) {
			System.out.println("-------- day " + i + " --------");
			System.out.println("name, sellIn, quality");
			for (Item item : items) {
				System.out.println(item);
			}
			System.out.println();
			app.updateQuality();
			Arrays.stream(items).map(item -> new Item(item.name, item.sellIn, item.quality)).forEach(itemList::add);
		}

		Approvals.verifyAll("items", itemList.toArray(new Item[0]));
	}

}