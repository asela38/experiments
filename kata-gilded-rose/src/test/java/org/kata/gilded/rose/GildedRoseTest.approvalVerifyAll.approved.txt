items[000] = +5 Dexterity Vest, 9, 19
items[001] = Aged Brie, 1, 1
items[002] = Elixir of the Mongoose, 4, 6
items[003] = Sulfuras, Hand of Ragnaros, 0, 80
items[004] = Sulfuras, Hand of Ragnaros, -1, 80
items[005] = Backstage passes to a TAFKAL80ETC concert, 14, 21
items[006] = Backstage passes to a TAFKAL80ETC concert, 9, 50
items[007] = Backstage passes to a TAFKAL80ETC concert, 4, 50
items[008] = Conjured Mana Cake, 2, 5
items[009] = +5 Dexterity Vest, 8, 18
items[010] = Aged Brie, 0, 2
items[011] = Elixir of the Mongoose, 3, 5
items[012] = Sulfuras, Hand of Ragnaros, 0, 80
items[013] = Sulfuras, Hand of Ragnaros, -1, 80
items[014] = Backstage passes to a TAFKAL80ETC concert, 13, 22
items[015] = Backstage passes to a TAFKAL80ETC concert, 8, 50
items[016] = Backstage passes to a TAFKAL80ETC concert, 3, 50
items[017] = Conjured Mana Cake, 1, 4
items[018] = +5 Dexterity Vest, 7, 17
items[019] = Aged Brie, -1, 4
items[020] = Elixir of the Mongoose, 2, 4
items[021] = Sulfuras, Hand of Ragnaros, 0, 80
items[022] = Sulfuras, Hand of Ragnaros, -1, 80
items[023] = Backstage passes to a TAFKAL80ETC concert, 12, 23
items[024] = Backstage passes to a TAFKAL80ETC concert, 7, 50
items[025] = Backstage passes to a TAFKAL80ETC concert, 2, 50
items[026] = Conjured Mana Cake, 0, 3
items[027] = +5 Dexterity Vest, 6, 16
items[028] = Aged Brie, -2, 6
items[029] = Elixir of the Mongoose, 1, 3
items[030] = Sulfuras, Hand of Ragnaros, 0, 80
items[031] = Sulfuras, Hand of Ragnaros, -1, 80
items[032] = Backstage passes to a TAFKAL80ETC concert, 11, 24
items[033] = Backstage passes to a TAFKAL80ETC concert, 6, 50
items[034] = Backstage passes to a TAFKAL80ETC concert, 1, 50
items[035] = Conjured Mana Cake, -1, 1
items[036] = +5 Dexterity Vest, 5, 15
items[037] = Aged Brie, -3, 8
items[038] = Elixir of the Mongoose, 0, 2
items[039] = Sulfuras, Hand of Ragnaros, 0, 80
items[040] = Sulfuras, Hand of Ragnaros, -1, 80
items[041] = Backstage passes to a TAFKAL80ETC concert, 10, 25
items[042] = Backstage passes to a TAFKAL80ETC concert, 5, 50
items[043] = Backstage passes to a TAFKAL80ETC concert, 0, 50
items[044] = Conjured Mana Cake, -2, 0
items[045] = +5 Dexterity Vest, 4, 14
items[046] = Aged Brie, -4, 10
items[047] = Elixir of the Mongoose, -1, 0
items[048] = Sulfuras, Hand of Ragnaros, 0, 80
items[049] = Sulfuras, Hand of Ragnaros, -1, 80
items[050] = Backstage passes to a TAFKAL80ETC concert, 9, 27
items[051] = Backstage passes to a TAFKAL80ETC concert, 4, 50
items[052] = Backstage passes to a TAFKAL80ETC concert, -1, 0
items[053] = Conjured Mana Cake, -3, 0
items[054] = +5 Dexterity Vest, 3, 13
items[055] = Aged Brie, -5, 12
items[056] = Elixir of the Mongoose, -2, 0
items[057] = Sulfuras, Hand of Ragnaros, 0, 80
items[058] = Sulfuras, Hand of Ragnaros, -1, 80
items[059] = Backstage passes to a TAFKAL80ETC concert, 8, 29
items[060] = Backstage passes to a TAFKAL80ETC concert, 3, 50
items[061] = Backstage passes to a TAFKAL80ETC concert, -2, 0
items[062] = Conjured Mana Cake, -4, 0
items[063] = +5 Dexterity Vest, 2, 12
items[064] = Aged Brie, -6, 14
items[065] = Elixir of the Mongoose, -3, 0
items[066] = Sulfuras, Hand of Ragnaros, 0, 80
items[067] = Sulfuras, Hand of Ragnaros, -1, 80
items[068] = Backstage passes to a TAFKAL80ETC concert, 7, 31
items[069] = Backstage passes to a TAFKAL80ETC concert, 2, 50
items[070] = Backstage passes to a TAFKAL80ETC concert, -3, 0
items[071] = Conjured Mana Cake, -5, 0
items[072] = +5 Dexterity Vest, 1, 11
items[073] = Aged Brie, -7, 16
items[074] = Elixir of the Mongoose, -4, 0
items[075] = Sulfuras, Hand of Ragnaros, 0, 80
items[076] = Sulfuras, Hand of Ragnaros, -1, 80
items[077] = Backstage passes to a TAFKAL80ETC concert, 6, 33
items[078] = Backstage passes to a TAFKAL80ETC concert, 1, 50
items[079] = Backstage passes to a TAFKAL80ETC concert, -4, 0
items[080] = Conjured Mana Cake, -6, 0
items[081] = +5 Dexterity Vest, 0, 10
items[082] = Aged Brie, -8, 18
items[083] = Elixir of the Mongoose, -5, 0
items[084] = Sulfuras, Hand of Ragnaros, 0, 80
items[085] = Sulfuras, Hand of Ragnaros, -1, 80
items[086] = Backstage passes to a TAFKAL80ETC concert, 5, 35
items[087] = Backstage passes to a TAFKAL80ETC concert, 0, 50
items[088] = Backstage passes to a TAFKAL80ETC concert, -5, 0
items[089] = Conjured Mana Cake, -7, 0
items[090] = +5 Dexterity Vest, -1, 8
items[091] = Aged Brie, -9, 20
items[092] = Elixir of the Mongoose, -6, 0
items[093] = Sulfuras, Hand of Ragnaros, 0, 80
items[094] = Sulfuras, Hand of Ragnaros, -1, 80
items[095] = Backstage passes to a TAFKAL80ETC concert, 4, 38
items[096] = Backstage passes to a TAFKAL80ETC concert, -1, 0
items[097] = Backstage passes to a TAFKAL80ETC concert, -6, 0
items[098] = Conjured Mana Cake, -8, 0
items[099] = +5 Dexterity Vest, -2, 6
items[100] = Aged Brie, -10, 22
items[101] = Elixir of the Mongoose, -7, 0
items[102] = Sulfuras, Hand of Ragnaros, 0, 80
items[103] = Sulfuras, Hand of Ragnaros, -1, 80
items[104] = Backstage passes to a TAFKAL80ETC concert, 3, 41
items[105] = Backstage passes to a TAFKAL80ETC concert, -2, 0
items[106] = Backstage passes to a TAFKAL80ETC concert, -7, 0
items[107] = Conjured Mana Cake, -9, 0
items[108] = +5 Dexterity Vest, -3, 4
items[109] = Aged Brie, -11, 24
items[110] = Elixir of the Mongoose, -8, 0
items[111] = Sulfuras, Hand of Ragnaros, 0, 80
items[112] = Sulfuras, Hand of Ragnaros, -1, 80
items[113] = Backstage passes to a TAFKAL80ETC concert, 2, 44
items[114] = Backstage passes to a TAFKAL80ETC concert, -3, 0
items[115] = Backstage passes to a TAFKAL80ETC concert, -8, 0
items[116] = Conjured Mana Cake, -10, 0
items[117] = +5 Dexterity Vest, -4, 2
items[118] = Aged Brie, -12, 26
items[119] = Elixir of the Mongoose, -9, 0
items[120] = Sulfuras, Hand of Ragnaros, 0, 80
items[121] = Sulfuras, Hand of Ragnaros, -1, 80
items[122] = Backstage passes to a TAFKAL80ETC concert, 1, 47
items[123] = Backstage passes to a TAFKAL80ETC concert, -4, 0
items[124] = Backstage passes to a TAFKAL80ETC concert, -9, 0
items[125] = Conjured Mana Cake, -11, 0
items[126] = +5 Dexterity Vest, -5, 0
items[127] = Aged Brie, -13, 28
items[128] = Elixir of the Mongoose, -10, 0
items[129] = Sulfuras, Hand of Ragnaros, 0, 80
items[130] = Sulfuras, Hand of Ragnaros, -1, 80
items[131] = Backstage passes to a TAFKAL80ETC concert, 0, 50
items[132] = Backstage passes to a TAFKAL80ETC concert, -5, 0
items[133] = Backstage passes to a TAFKAL80ETC concert, -10, 0
items[134] = Conjured Mana Cake, -12, 0
items[135] = +5 Dexterity Vest, -6, 0
items[136] = Aged Brie, -14, 30
items[137] = Elixir of the Mongoose, -11, 0
items[138] = Sulfuras, Hand of Ragnaros, 0, 80
items[139] = Sulfuras, Hand of Ragnaros, -1, 80
items[140] = Backstage passes to a TAFKAL80ETC concert, -1, 0
items[141] = Backstage passes to a TAFKAL80ETC concert, -6, 0
items[142] = Backstage passes to a TAFKAL80ETC concert, -11, 0
items[143] = Conjured Mana Cake, -13, 0
items[144] = +5 Dexterity Vest, -7, 0
items[145] = Aged Brie, -15, 32
items[146] = Elixir of the Mongoose, -12, 0
items[147] = Sulfuras, Hand of Ragnaros, 0, 80
items[148] = Sulfuras, Hand of Ragnaros, -1, 80
items[149] = Backstage passes to a TAFKAL80ETC concert, -2, 0
items[150] = Backstage passes to a TAFKAL80ETC concert, -7, 0
items[151] = Backstage passes to a TAFKAL80ETC concert, -12, 0
items[152] = Conjured Mana Cake, -14, 0
items[153] = +5 Dexterity Vest, -8, 0
items[154] = Aged Brie, -16, 34
items[155] = Elixir of the Mongoose, -13, 0
items[156] = Sulfuras, Hand of Ragnaros, 0, 80
items[157] = Sulfuras, Hand of Ragnaros, -1, 80
items[158] = Backstage passes to a TAFKAL80ETC concert, -3, 0
items[159] = Backstage passes to a TAFKAL80ETC concert, -8, 0
items[160] = Backstage passes to a TAFKAL80ETC concert, -13, 0
items[161] = Conjured Mana Cake, -15, 0
items[162] = +5 Dexterity Vest, -9, 0
items[163] = Aged Brie, -17, 36
items[164] = Elixir of the Mongoose, -14, 0
items[165] = Sulfuras, Hand of Ragnaros, 0, 80
items[166] = Sulfuras, Hand of Ragnaros, -1, 80
items[167] = Backstage passes to a TAFKAL80ETC concert, -4, 0
items[168] = Backstage passes to a TAFKAL80ETC concert, -9, 0
items[169] = Backstage passes to a TAFKAL80ETC concert, -14, 0
items[170] = Conjured Mana Cake, -16, 0
items[171] = +5 Dexterity Vest, -10, 0
items[172] = Aged Brie, -18, 38
items[173] = Elixir of the Mongoose, -15, 0
items[174] = Sulfuras, Hand of Ragnaros, 0, 80
items[175] = Sulfuras, Hand of Ragnaros, -1, 80
items[176] = Backstage passes to a TAFKAL80ETC concert, -5, 0
items[177] = Backstage passes to a TAFKAL80ETC concert, -10, 0
items[178] = Backstage passes to a TAFKAL80ETC concert, -15, 0
items[179] = Conjured Mana Cake, -17, 0
items[180] = +5 Dexterity Vest, -11, 0
items[181] = Aged Brie, -19, 40
items[182] = Elixir of the Mongoose, -16, 0
items[183] = Sulfuras, Hand of Ragnaros, 0, 80
items[184] = Sulfuras, Hand of Ragnaros, -1, 80
items[185] = Backstage passes to a TAFKAL80ETC concert, -6, 0
items[186] = Backstage passes to a TAFKAL80ETC concert, -11, 0
items[187] = Backstage passes to a TAFKAL80ETC concert, -16, 0
items[188] = Conjured Mana Cake, -18, 0
items[189] = +5 Dexterity Vest, -12, 0
items[190] = Aged Brie, -20, 42
items[191] = Elixir of the Mongoose, -17, 0
items[192] = Sulfuras, Hand of Ragnaros, 0, 80
items[193] = Sulfuras, Hand of Ragnaros, -1, 80
items[194] = Backstage passes to a TAFKAL80ETC concert, -7, 0
items[195] = Backstage passes to a TAFKAL80ETC concert, -12, 0
items[196] = Backstage passes to a TAFKAL80ETC concert, -17, 0
items[197] = Conjured Mana Cake, -19, 0
items[198] = +5 Dexterity Vest, -13, 0
items[199] = Aged Brie, -21, 44
items[200] = Elixir of the Mongoose, -18, 0
items[201] = Sulfuras, Hand of Ragnaros, 0, 80
items[202] = Sulfuras, Hand of Ragnaros, -1, 80
items[203] = Backstage passes to a TAFKAL80ETC concert, -8, 0
items[204] = Backstage passes to a TAFKAL80ETC concert, -13, 0
items[205] = Backstage passes to a TAFKAL80ETC concert, -18, 0
items[206] = Conjured Mana Cake, -20, 0
items[207] = +5 Dexterity Vest, -14, 0
items[208] = Aged Brie, -22, 46
items[209] = Elixir of the Mongoose, -19, 0
items[210] = Sulfuras, Hand of Ragnaros, 0, 80
items[211] = Sulfuras, Hand of Ragnaros, -1, 80
items[212] = Backstage passes to a TAFKAL80ETC concert, -9, 0
items[213] = Backstage passes to a TAFKAL80ETC concert, -14, 0
items[214] = Backstage passes to a TAFKAL80ETC concert, -19, 0
items[215] = Conjured Mana Cake, -21, 0
items[216] = +5 Dexterity Vest, -15, 0
items[217] = Aged Brie, -23, 48
items[218] = Elixir of the Mongoose, -20, 0
items[219] = Sulfuras, Hand of Ragnaros, 0, 80
items[220] = Sulfuras, Hand of Ragnaros, -1, 80
items[221] = Backstage passes to a TAFKAL80ETC concert, -10, 0
items[222] = Backstage passes to a TAFKAL80ETC concert, -15, 0
items[223] = Backstage passes to a TAFKAL80ETC concert, -20, 0
items[224] = Conjured Mana Cake, -22, 0
items[225] = +5 Dexterity Vest, -16, 0
items[226] = Aged Brie, -24, 50
items[227] = Elixir of the Mongoose, -21, 0
items[228] = Sulfuras, Hand of Ragnaros, 0, 80
items[229] = Sulfuras, Hand of Ragnaros, -1, 80
items[230] = Backstage passes to a TAFKAL80ETC concert, -11, 0
items[231] = Backstage passes to a TAFKAL80ETC concert, -16, 0
items[232] = Backstage passes to a TAFKAL80ETC concert, -21, 0
items[233] = Conjured Mana Cake, -23, 0
items[234] = +5 Dexterity Vest, -17, 0
items[235] = Aged Brie, -25, 50
items[236] = Elixir of the Mongoose, -22, 0
items[237] = Sulfuras, Hand of Ragnaros, 0, 80
items[238] = Sulfuras, Hand of Ragnaros, -1, 80
items[239] = Backstage passes to a TAFKAL80ETC concert, -12, 0
items[240] = Backstage passes to a TAFKAL80ETC concert, -17, 0
items[241] = Backstage passes to a TAFKAL80ETC concert, -22, 0
items[242] = Conjured Mana Cake, -24, 0
items[243] = +5 Dexterity Vest, -18, 0
items[244] = Aged Brie, -26, 50
items[245] = Elixir of the Mongoose, -23, 0
items[246] = Sulfuras, Hand of Ragnaros, 0, 80
items[247] = Sulfuras, Hand of Ragnaros, -1, 80
items[248] = Backstage passes to a TAFKAL80ETC concert, -13, 0
items[249] = Backstage passes to a TAFKAL80ETC concert, -18, 0
items[250] = Backstage passes to a TAFKAL80ETC concert, -23, 0
items[251] = Conjured Mana Cake, -25, 0
items[252] = +5 Dexterity Vest, -19, 0
items[253] = Aged Brie, -27, 50
items[254] = Elixir of the Mongoose, -24, 0
items[255] = Sulfuras, Hand of Ragnaros, 0, 80
items[256] = Sulfuras, Hand of Ragnaros, -1, 80
items[257] = Backstage passes to a TAFKAL80ETC concert, -14, 0
items[258] = Backstage passes to a TAFKAL80ETC concert, -19, 0
items[259] = Backstage passes to a TAFKAL80ETC concert, -24, 0
items[260] = Conjured Mana Cake, -26, 0
items[261] = +5 Dexterity Vest, -20, 0
items[262] = Aged Brie, -28, 50
items[263] = Elixir of the Mongoose, -25, 0
items[264] = Sulfuras, Hand of Ragnaros, 0, 80
items[265] = Sulfuras, Hand of Ragnaros, -1, 80
items[266] = Backstage passes to a TAFKAL80ETC concert, -15, 0
items[267] = Backstage passes to a TAFKAL80ETC concert, -20, 0
items[268] = Backstage passes to a TAFKAL80ETC concert, -25, 0
items[269] = Conjured Mana Cake, -27, 0
items[270] = +5 Dexterity Vest, -21, 0
items[271] = Aged Brie, -29, 50
items[272] = Elixir of the Mongoose, -26, 0
items[273] = Sulfuras, Hand of Ragnaros, 0, 80
items[274] = Sulfuras, Hand of Ragnaros, -1, 80
items[275] = Backstage passes to a TAFKAL80ETC concert, -16, 0
items[276] = Backstage passes to a TAFKAL80ETC concert, -21, 0
items[277] = Backstage passes to a TAFKAL80ETC concert, -26, 0
items[278] = Conjured Mana Cake, -28, 0
items[279] = +5 Dexterity Vest, -22, 0
items[280] = Aged Brie, -30, 50
items[281] = Elixir of the Mongoose, -27, 0
items[282] = Sulfuras, Hand of Ragnaros, 0, 80
items[283] = Sulfuras, Hand of Ragnaros, -1, 80
items[284] = Backstage passes to a TAFKAL80ETC concert, -17, 0
items[285] = Backstage passes to a TAFKAL80ETC concert, -22, 0
items[286] = Backstage passes to a TAFKAL80ETC concert, -27, 0
items[287] = Conjured Mana Cake, -29, 0
items[288] = +5 Dexterity Vest, -23, 0
items[289] = Aged Brie, -31, 50
items[290] = Elixir of the Mongoose, -28, 0
items[291] = Sulfuras, Hand of Ragnaros, 0, 80
items[292] = Sulfuras, Hand of Ragnaros, -1, 80
items[293] = Backstage passes to a TAFKAL80ETC concert, -18, 0
items[294] = Backstage passes to a TAFKAL80ETC concert, -23, 0
items[295] = Backstage passes to a TAFKAL80ETC concert, -28, 0
items[296] = Conjured Mana Cake, -30, 0
items[297] = +5 Dexterity Vest, -24, 0
items[298] = Aged Brie, -32, 50
items[299] = Elixir of the Mongoose, -29, 0
items[300] = Sulfuras, Hand of Ragnaros, 0, 80
items[301] = Sulfuras, Hand of Ragnaros, -1, 80
items[302] = Backstage passes to a TAFKAL80ETC concert, -19, 0
items[303] = Backstage passes to a TAFKAL80ETC concert, -24, 0
items[304] = Backstage passes to a TAFKAL80ETC concert, -29, 0
items[305] = Conjured Mana Cake, -31, 0
items[306] = +5 Dexterity Vest, -25, 0
items[307] = Aged Brie, -33, 50
items[308] = Elixir of the Mongoose, -30, 0
items[309] = Sulfuras, Hand of Ragnaros, 0, 80
items[310] = Sulfuras, Hand of Ragnaros, -1, 80
items[311] = Backstage passes to a TAFKAL80ETC concert, -20, 0
items[312] = Backstage passes to a TAFKAL80ETC concert, -25, 0
items[313] = Backstage passes to a TAFKAL80ETC concert, -30, 0
items[314] = Conjured Mana Cake, -32, 0
items[315] = +5 Dexterity Vest, -26, 0
items[316] = Aged Brie, -34, 50
items[317] = Elixir of the Mongoose, -31, 0
items[318] = Sulfuras, Hand of Ragnaros, 0, 80
items[319] = Sulfuras, Hand of Ragnaros, -1, 80
items[320] = Backstage passes to a TAFKAL80ETC concert, -21, 0
items[321] = Backstage passes to a TAFKAL80ETC concert, -26, 0
items[322] = Backstage passes to a TAFKAL80ETC concert, -31, 0
items[323] = Conjured Mana Cake, -33, 0
items[324] = +5 Dexterity Vest, -27, 0
items[325] = Aged Brie, -35, 50
items[326] = Elixir of the Mongoose, -32, 0
items[327] = Sulfuras, Hand of Ragnaros, 0, 80
items[328] = Sulfuras, Hand of Ragnaros, -1, 80
items[329] = Backstage passes to a TAFKAL80ETC concert, -22, 0
items[330] = Backstage passes to a TAFKAL80ETC concert, -27, 0
items[331] = Backstage passes to a TAFKAL80ETC concert, -32, 0
items[332] = Conjured Mana Cake, -34, 0
items[333] = +5 Dexterity Vest, -28, 0
items[334] = Aged Brie, -36, 50
items[335] = Elixir of the Mongoose, -33, 0
items[336] = Sulfuras, Hand of Ragnaros, 0, 80
items[337] = Sulfuras, Hand of Ragnaros, -1, 80
items[338] = Backstage passes to a TAFKAL80ETC concert, -23, 0
items[339] = Backstage passes to a TAFKAL80ETC concert, -28, 0
items[340] = Backstage passes to a TAFKAL80ETC concert, -33, 0
items[341] = Conjured Mana Cake, -35, 0
items[342] = +5 Dexterity Vest, -29, 0
items[343] = Aged Brie, -37, 50
items[344] = Elixir of the Mongoose, -34, 0
items[345] = Sulfuras, Hand of Ragnaros, 0, 80
items[346] = Sulfuras, Hand of Ragnaros, -1, 80
items[347] = Backstage passes to a TAFKAL80ETC concert, -24, 0
items[348] = Backstage passes to a TAFKAL80ETC concert, -29, 0
items[349] = Backstage passes to a TAFKAL80ETC concert, -34, 0
items[350] = Conjured Mana Cake, -36, 0
items[351] = +5 Dexterity Vest, -30, 0
items[352] = Aged Brie, -38, 50
items[353] = Elixir of the Mongoose, -35, 0
items[354] = Sulfuras, Hand of Ragnaros, 0, 80
items[355] = Sulfuras, Hand of Ragnaros, -1, 80
items[356] = Backstage passes to a TAFKAL80ETC concert, -25, 0
items[357] = Backstage passes to a TAFKAL80ETC concert, -30, 0
items[358] = Backstage passes to a TAFKAL80ETC concert, -35, 0
items[359] = Conjured Mana Cake, -37, 0
items[360] = +5 Dexterity Vest, -31, 0
items[361] = Aged Brie, -39, 50
items[362] = Elixir of the Mongoose, -36, 0
items[363] = Sulfuras, Hand of Ragnaros, 0, 80
items[364] = Sulfuras, Hand of Ragnaros, -1, 80
items[365] = Backstage passes to a TAFKAL80ETC concert, -26, 0
items[366] = Backstage passes to a TAFKAL80ETC concert, -31, 0
items[367] = Backstage passes to a TAFKAL80ETC concert, -36, 0
items[368] = Conjured Mana Cake, -38, 0
items[369] = +5 Dexterity Vest, -32, 0
items[370] = Aged Brie, -40, 50
items[371] = Elixir of the Mongoose, -37, 0
items[372] = Sulfuras, Hand of Ragnaros, 0, 80
items[373] = Sulfuras, Hand of Ragnaros, -1, 80
items[374] = Backstage passes to a TAFKAL80ETC concert, -27, 0
items[375] = Backstage passes to a TAFKAL80ETC concert, -32, 0
items[376] = Backstage passes to a TAFKAL80ETC concert, -37, 0
items[377] = Conjured Mana Cake, -39, 0
items[378] = +5 Dexterity Vest, -33, 0
items[379] = Aged Brie, -41, 50
items[380] = Elixir of the Mongoose, -38, 0
items[381] = Sulfuras, Hand of Ragnaros, 0, 80
items[382] = Sulfuras, Hand of Ragnaros, -1, 80
items[383] = Backstage passes to a TAFKAL80ETC concert, -28, 0
items[384] = Backstage passes to a TAFKAL80ETC concert, -33, 0
items[385] = Backstage passes to a TAFKAL80ETC concert, -38, 0
items[386] = Conjured Mana Cake, -40, 0
items[387] = +5 Dexterity Vest, -34, 0
items[388] = Aged Brie, -42, 50
items[389] = Elixir of the Mongoose, -39, 0
items[390] = Sulfuras, Hand of Ragnaros, 0, 80
items[391] = Sulfuras, Hand of Ragnaros, -1, 80
items[392] = Backstage passes to a TAFKAL80ETC concert, -29, 0
items[393] = Backstage passes to a TAFKAL80ETC concert, -34, 0
items[394] = Backstage passes to a TAFKAL80ETC concert, -39, 0
items[395] = Conjured Mana Cake, -41, 0
items[396] = +5 Dexterity Vest, -35, 0
items[397] = Aged Brie, -43, 50
items[398] = Elixir of the Mongoose, -40, 0
items[399] = Sulfuras, Hand of Ragnaros, 0, 80
items[400] = Sulfuras, Hand of Ragnaros, -1, 80
items[401] = Backstage passes to a TAFKAL80ETC concert, -30, 0
items[402] = Backstage passes to a TAFKAL80ETC concert, -35, 0
items[403] = Backstage passes to a TAFKAL80ETC concert, -40, 0
items[404] = Conjured Mana Cake, -42, 0
items[405] = +5 Dexterity Vest, -36, 0
items[406] = Aged Brie, -44, 50
items[407] = Elixir of the Mongoose, -41, 0
items[408] = Sulfuras, Hand of Ragnaros, 0, 80
items[409] = Sulfuras, Hand of Ragnaros, -1, 80
items[410] = Backstage passes to a TAFKAL80ETC concert, -31, 0
items[411] = Backstage passes to a TAFKAL80ETC concert, -36, 0
items[412] = Backstage passes to a TAFKAL80ETC concert, -41, 0
items[413] = Conjured Mana Cake, -43, 0
items[414] = +5 Dexterity Vest, -37, 0
items[415] = Aged Brie, -45, 50
items[416] = Elixir of the Mongoose, -42, 0
items[417] = Sulfuras, Hand of Ragnaros, 0, 80
items[418] = Sulfuras, Hand of Ragnaros, -1, 80
items[419] = Backstage passes to a TAFKAL80ETC concert, -32, 0
items[420] = Backstage passes to a TAFKAL80ETC concert, -37, 0
items[421] = Backstage passes to a TAFKAL80ETC concert, -42, 0
items[422] = Conjured Mana Cake, -44, 0
items[423] = +5 Dexterity Vest, -38, 0
items[424] = Aged Brie, -46, 50
items[425] = Elixir of the Mongoose, -43, 0
items[426] = Sulfuras, Hand of Ragnaros, 0, 80
items[427] = Sulfuras, Hand of Ragnaros, -1, 80
items[428] = Backstage passes to a TAFKAL80ETC concert, -33, 0
items[429] = Backstage passes to a TAFKAL80ETC concert, -38, 0
items[430] = Backstage passes to a TAFKAL80ETC concert, -43, 0
items[431] = Conjured Mana Cake, -45, 0
items[432] = +5 Dexterity Vest, -39, 0
items[433] = Aged Brie, -47, 50
items[434] = Elixir of the Mongoose, -44, 0
items[435] = Sulfuras, Hand of Ragnaros, 0, 80
items[436] = Sulfuras, Hand of Ragnaros, -1, 80
items[437] = Backstage passes to a TAFKAL80ETC concert, -34, 0
items[438] = Backstage passes to a TAFKAL80ETC concert, -39, 0
items[439] = Backstage passes to a TAFKAL80ETC concert, -44, 0
items[440] = Conjured Mana Cake, -46, 0
items[441] = +5 Dexterity Vest, -40, 0
items[442] = Aged Brie, -48, 50
items[443] = Elixir of the Mongoose, -45, 0
items[444] = Sulfuras, Hand of Ragnaros, 0, 80
items[445] = Sulfuras, Hand of Ragnaros, -1, 80
items[446] = Backstage passes to a TAFKAL80ETC concert, -35, 0
items[447] = Backstage passes to a TAFKAL80ETC concert, -40, 0
items[448] = Backstage passes to a TAFKAL80ETC concert, -45, 0
items[449] = Conjured Mana Cake, -47, 0
